docs/book/src/developers/cluster-creation.md

```diff
@@ -0,0 +1,33 @@
+# Creating cluster without clusterctl
+
+This document describes how to create a management cluster and workload cluster without using clusterctl.
+For creating a cluster with clusterctl, checkout our [Cluster API Quick Start](https://cluster-api.sigs.k8s.io/user/quick-start.html)
+
+## For creating a Management cluster
+
+1. Build required images by using the following commands:
+   
+   - `docker pull gcr.io/k8s-staging-cluster-api-gcp/cluster-api-gcp-controller:v20211110-v0.4.0-18-gba203a6a`
```

Comment:
why you pull this image? i guess this is not required

Reply:
I did this for the controller image actually... I will remove this